src/avatar/__tests__/Avatar.js

```diff
@@ -204,5 +204,20 @@ describe('Avatar Component', () => {
       expect(component.props().style.backgroundColor).toBe('transparent');
       expect(toJson(component)).toMatchSnapshot();
     });
+
+    it("shouldn't show placeholder if source doesn't contain uri property", () => {
+      const component = shallow(
+        <Avatar
+          size="medium"
+          overlayContainerStyle={{ backgroundColor: 'pink' }}
+          title="MD"
+          source={{}}
+        />
+      );
+      expect(
+        component.children().props().placeholderStyle.backgroundColor
+      ).toBe('transparent');
+      expect(toJson(component)).toMatchSnapshot();
+    });
   });
```

Comment:
Hi @jehielmartinez this looks great. Can you pls add the test case when source.uri is undefined/.

`source = {{  uri : undefined }}`

Reply:
sure!